src/BrightScriptCommands.ts

```diff
@@ -188,6 +252,14 @@ export class BrightScriptCommands {
         }
     }
 
+    private registerCommand(name: string, callback: (...args: any[]) => any, thisArg?: any) {
```

Comment:
Just broke it out to clean up the main register function as it was getting a bit noisy.